src/core/components/AnimationWrapperView.tsx

```diff
@@ -35,14 +35,15 @@ export default class AnimationWrapperView extends React.Component<AnimationWrapp
 
     public render(): React.ReactNode | undefined {
         this._assertChildType();
-        const { animationConfig, children, onAnimationFinish, onAnimationStart } = this.props;
+        const { animationConfig, children, onAnimationFinish, onAnimationStart, animationWrapperStyles } = this.props;
         if (this._animationComponentClass && children) {
             return (
                 <this._animationComponentClass
                     ref={this._setRef}
                     animationConfig={animationConfig as any}
                     onAnimationFinish={onAnimationFinish}
-                    onAnimationStart={onAnimationStart}>
+                    onAnimationStart={onAnimationStart}
+                    animationWrapperStyles={animationWrapperStyles}>
```

Comment:
can you update the documentation of this newly added prop?

Reply:
If this is only pertaining to the JsonAnimationWrapper, we should pass it in JsonAnimationConfig, and consume it in the JsonAnimationWrapper

Reply:
It's not related to only `JsonAnimationWrapper`, it will be extended to other soon.

Reply:
readme changed